src/entity_embedding_utils.jl

```diff
@@ -125,7 +127,8 @@ function MLJModelInterface.transform(
     fitresult,
     Xnew,
 )
```

Comment:
**Important:** We need `transformer::MLJFluxModel` here or this is ambiguous.

Reply:
OK.